.ci_support/requirements.yaml

```diff
@@ -0,0 +1,13 @@
+name: staged-recipes
```

Comment:
Having the deps here and in the .toml is not ideal. Any way around this?

Reply:
Let's add linting for now in case we need to roll back the pixi stuff.

Reply:
You can do:
```
pixi project export conda-environment > requirements.yaml
```
Maybe that helps out here.

Reply:
Oh wow, indeed! Thanks for the pointer!